playground/nextjs-app-router/components/demo/Wallet.tsx

```diff
@@ -28,11 +29,12 @@ function WalletComponent() {
           <Name />
         </ConnectWallet>
         <WalletDropdown>
-          <Identity className="px-4 pt-3 pb-2" hasCopyAddressOnClick={true}>
```

Comment:
?

Reply:
Updated the Identity component to have the Address be clickable rather than the entire Identity component. With the addition of the icons, there are multiple elements that can be clicked on. In a follow up PR I will deprecate this field entirely

Reply:
you may just be able to stopPropagation on your link clicks and keep this functionality intact